frontend/app/components/input-checkboxes.tsx

```diff
@@ -0,0 +1,72 @@
+import { InputError } from './input-error';
+import { InputHelp } from './input-help';
+import { InputLegend } from '~/components/input-legend';
+import { cn } from '~/utils/tw-utils';
+
+export interface InputCheckboxesProps {
+  errorMessage?: string;
+  helpMessagePrimary?: React.ReactNode;
+  helpMessagePrimaryClassName?: string;
+  helpMessageSecondary?: React.ReactNode;
+  helpMessageSecondaryClassName?: string;
+  id: string;
+  options: Omit<ComponentProps<typeof InputCheckbox>, 'aria-describedby' | 'aria-errormessage' | 'aria-invalid' | 'aria-required' | 'id' | 'name' | 'required'>[];
+  legend: ReactNode;
+  name: string;
+  required?: boolean;
+}
+
+const InputCheckboxes = ({ errorMessage, helpMessagePrimary, helpMessagePrimaryClassName, helpMessageSecondary, helpMessageSecondaryClassName, id, legend, name, options, required }: InputCheckboxesProps) => {
+  const inputErrorId = `input-checkboxes-${id}-error`;
+  const inputHelpMessagePrimaryId = `input-checkboxes-${id}-help-primary`;
+  const inputHelpMessageSecondaryId = `input-checkboxes-${id}-help-secondary`;
+  const inputLegendId = `input-checkboxes-${id}-legend`;
+  const inputWrapperId = `input-checkboxes-${id}`;
+
+  function getAriaDescribedby() {
+    const ariaDescribedby = [inputLegendId];
+    if (helpMessagePrimary) ariaDescribedby.push(inputHelpMessagePrimaryId);
+    if (helpMessageSecondary) ariaDescribedby.push(inputHelpMessageSecondaryId);
+    return ariaDescribedby.length > 0 ? ariaDescribedby.join(' ') : undefined;
+  }
+
+  return (
+    <fieldset id={inputWrapperId} data-testid={inputWrapperId}>
+      <InputLegend id={inputLegendId} required={required} className="mb-2">
+        {legend}
+      </InputLegend>
+      {helpMessagePrimary && (
+        <InputHelp id={inputHelpMessagePrimaryId} className={cn('mb-2', helpMessagePrimaryClassName)} data-testid="input-field-help-primary">
+          {helpMessagePrimary}
+        </InputHelp>
+      )}
+      <ul className="space-y-2">
+        {options.map((option, index) => {
+          const inputCheckboxId = `${id}-option-${index}`;
+          return (
+            <li key={inputCheckboxId}>
+              <InputCheckbox aria-describedby={getAriaDescribedby()} aria-errormessage={errorMessage && inputErrorId} aria-invalid={!!errorMessage} aria-required={required} id={inputCheckboxId} name={name} required={required} {...option} />
+            </li>
+          );
+        })}
+      </ul>
+      {errorMessage && (
+        <InputError id={inputErrorId} className="mt-2">
+          {errorMessage}
+        </InputError>
+      )}
+      {helpMessageSecondary && (
+        <InputHelp id={inputHelpMessageSecondaryId} className={cn('mt-2', helpMessageSecondaryClassName)} data-testid="input-field-help-secondary">
+          {helpMessageSecondary}
+        </InputHelp>
+      )}
+    </fieldset>
+  );
+};
+
+InputCheckboxes.displayName = 'InputCheckboxes';
+
+export { InputCheckboxes };
```

Comment:
This can be removed once you switch to `export function()` as per previous comment.

Reply:
Implemented in commit dec2ceef9218bed7957453837c8f49c7cb1d2692.